donate/forms.py

```diff
@@ -218,10 +220,37 @@ class Meta(object):
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.fields['user'].widget = forms.HiddenInput()
-        pass
 
         
 class SchoolDonationForm(forms.ModelForm):
     class Meta:
         model = SchoolDonation
         fields = ['name', 'email', 'contact', 'state', 'city', 'address', 'amount', 'note']
+
+class AcademicSubscriptionForm(forms.Form):
+    institute = forms.ModelMultipleChoiceField(queryset=AcademicCenter.objects.all().order_by('institution_name'))
+    state = forms.ModelChoiceField(queryset=State.objects.all().order_by('name'),
+                                   empty_label='----Select State----')
+    email = forms.EmailField(validators=[EmailValidator(message="Enter a valid email address.")], required=False)
+    name = forms.CharField(max_length=255)
+    phone = forms.CharField(max_length=10)
```

Comment:
Max length is 10. Does this mean only Indian numbers are allowed here? See comment below also.